ethereum/permissioning/src/main/java/org/hyperledger/besu/ethereum/permissioning/account/AccountPermissioningController.java

```diff
@@ -75,6 +80,16 @@ public boolean isPermitted(
     final boolean permitted = permittedLocal && permittedOnchain;
 
     if (permitted) {
+      for (TransactionPermissioningProvider provider : this.pluginProviders) {
+        if (!provider.isPermitted(transaction)) {
+          LOG.trace(
+              "Account permissioning - {}: Rejected transaction {} from {}",
```

Comment:
thinking we should standardise the class naming and logging to "Transaction permissioning" rather than "Account permissioning" - but this can be a separate PR